experiments/tf_trainer/common/tfrecord_simple.py

```diff
@@ -10,6 +10,22 @@
 from tf_trainer.common import dataset_input
 from tf_trainer.common import types
 from typing import Dict
+from typing import List
+
+
+tf.app.flags.DEFINE_string('train_path', None,
```

Comment:
I actually think this part of the PR change - of making these global re-used flags - is net worse for the codebase. I know it puts some common code in one place, but it pushes the flag var definitions throughout many files, so to know the flags you have to do more search, and it's not obvious from the run files how far you have to dig, and what you have to provide. If we want to abstract these, I think it's best to do it into a file called `common_arguments.py` or something like that, so that it is directly included in the run files. What do you think?

Reply:
Why I disagree: the flags are now only scoped to this single file, they are no longer globally re-used, or at lest that is my intention. That is why I introduced accessors to the input classes to catch attempts to access flag contents.

That makes it possible to introduce a new abstraction that maps say 'civil_comments' -> input files, validation file, label keys etc.

But: that can't be done at the same step because we have about 12 shell scripts that all specify flags.

This is a behavior preserving change. You suggestion would broaden the scope and require changes to multiple files for which there is no test coverage. More importantly, the idea of common_arguments belies the more fundamental problem with our abstractions that two file names and a list of strings are inadequate to describe more than a subset of potential inputs.

Reply:
OK, discussed and agreed that we want examples to clearly explain usage as we move flags out of the run.py, so that it's clear what the key flags are. And we will followup with some large improvements to structure of data input in future.